packages/contractkit/src/kit.ts

```diff
@@ -67,6 +73,38 @@ export class ContractKit {
     return toTxResult(promiEvent)
   }
 
+  async sendSignedTransaction(tx: Tx): Promise<TransactionResult> {
+    const gasFeeRecipient = await this.web3.eth.getCoinbase()
```

Comment:
this should happen on the provider, not at `kit` level.
so, if i use web3 directly, i can still send a tx

Reply:
forget about this.... can happen either place! :P

Reply:
Added them to `signTransactionAsync`. Also, removed `sendSignedTransaction` and added `gasFeeRecipient` calculation to `sendTransaction`. This makes that method async though.